mcoding_bot/cogs/owner.py

```diff
@@ -23,6 +23,7 @@ async def gitpull(self, ctx: pincer.objects.MessageContext):
         if ctx.author.id not in self.bot.config.owner_ids:
             return "You don't have permission to do this."
         os.system("git pull")
+        await ctx.reply("Pulled from github, restarting bot...")
```

Comment:
Would be nice to have answer after the pull

Reply:
It is after the pull

Reply:
Though we probably need to defer so it doesn't timeout before the pull

Reply:
to avoid timeout use `await ctx.ack()` at top of the func

Reply:
Done